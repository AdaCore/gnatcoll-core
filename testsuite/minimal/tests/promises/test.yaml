description: Basic test for GNATCOLL.Promises.
