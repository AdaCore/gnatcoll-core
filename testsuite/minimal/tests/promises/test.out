=== Create chain
Resolving...
Convert_Int.On_Next  input= 2
Convert_Float.On_Next  input= 2.00000E+00
Display_String.On_Next  input=value was 2.00000E+00
Display_String.On_Next  input=value was 2.00000E+00
Display_Int.On_Next  input= 2
=== Create chain, will Fail
Failing...
Display_String.Failed because Explicit failure
=== Create chain, will Fail in middle
Convert_Int.On_Next  input= 3
Fail_On_Float: mark output as failed
Display_String.Failed because explicit
Display_String.Failed because explicit
